topostats/default_config.yaml

```diff
@@ -33,17 +33,10 @@ grains:
   # Thresholding by height
   grain_crop_padding: 1 # Padding to apply to grains. Needs to be at least 1, more padding may help with unets.
   threshold_method: std_dev # Options : std_dev, otsu, absolute, unet
-  otsu_threshold_multiplier: 1.0
-  threshold_std_dev:
-    below: [10.0] # Thresholds for grains below the image background. List[float].
-    above: [1.0] # Thresholds for grains above the image background. List[float].
-  threshold_absolute:
-    below: [-1.0] # Thresholds for grains below the image background. List[float].
-    above: [1.0] # Thresholds for grains above the image background. List[float].
-  direction: above # Options: above, below, both (defines whether to look for grains above or below thresholds or both)
-  area_thresholds:
-    above: [300, 3000] # above surface [Low, High] in nm^2 (also takes null)
-    below: [null, null] # below surface [Low, High] in nm^2 (also takes null)
+  otsu_threshold_multiplier: 1.0 # Thresholds for data
+  threshold_std_dev: [1.0] # Thresholds for data either side of the image background, positive for above and negative for below
```

Comment:
Does this need to be a list? Would people ever want to use thresholds that are different standard deviations for each direction? If not it can just be a float and when `below` is required its multiplied by `-1`.

This impacts the `Grains` class and the type-hints used there.

How many users use standard deviation thresholding and of those how many and how often use different standard deviations for directions?

`<aside>`

As a statistician it jars/is confusing to see reference to a negative standard deviation since it is always >= 0 being the square root of the variance!
`</aside>`